packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -479,6 +479,7 @@ export = {
         source: codebuild.Source.gitHub({
           owner: 'testowner',
           repo: 'testrepo',
+          branch: 'testbranch',
```

Comment:
I think the test you added below is fine - no need for this one :).

Reply:
👍 Sounds good. I removed this test.